### PR TITLE
sleeping and syncing to fix issues with docker hub builds

### LINKED ISSUE #374: 3.2.4 causing issues with custom docker builds?
No idea if this is related, but I have automated docker builds configured to trigger whenever there is a new `gliderlabs/logspout` docker image pushed to docker hub.

My image builds from https://github.com/gliderlabs/logspout/blob/master/custom/Dockerfile on docker hub via automated builds.  It works on my local, but is failing on docker hub now:

```
Building in Docker Cloud's infrastructure...
Cloning into '.'...

KernelVersion: 4.4.0-93-generic
Arch: amd64
BuildTime: 2017-08-17T22:50:04.828747906+00:00
ApiVersion: 1.30
Version: 17.06.1-ce
MinAPIVersion: 1.12
GitCommit: 874a737
Os: linux
GoVersion: go1.8.3
Starting build of index.docker.io/billimek/logspout:gelf...
Step 1/2 : FROM gliderlabs/logspout:master

[91m# Executing 3 build triggers...
[0m
Step 1/1 : COPY ./build.sh /src/build.sh

Step 1/1 : COPY ./modules.go /src/modules.go

Step 1/1 : RUN cd /src && chmod +x ./build.sh && ./build.sh "$(cat VERSION)-custom"

 ---> Running in f395c450159e

[91m/bin/sh: ./build.sh: Text file busy
[0m
Removing intermediate container 75197530001b

Removing intermediate container f395c450159e

Removing intermediate container 0eda008907ff

The command '/bin/sh -c cd /src && chmod +x ./build.sh && ./build.sh "$(cat VERSION)-custom"' returned a non-zero code: 2
```

[build log](https://hub.docker.com/r/billimek/logspout/builds/bwa9fql7whhkcbegjpbqtnk/).

Given that the error is about `./build.sh: Text file busy` I wonder if the change to `chmod +x ./build.sh` is somehow inexplicably causing this failure?

### PR DESCRIPTION
**Pull request by billimek**
Should fix #374

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ RUN cd /src && ./build.sh "$(cat VERSION)"
 
 ONBUILD COPY ./build.sh /src/build.sh
 ONBUILD COPY ./modules.go /src/modules.go
-ONBUILD RUN cd /src && chmod +x ./build.sh && ./build.sh "$(cat VERSION)-custom"
+ONBUILD RUN cd /src && chmod +x ./build.sh ; sleep 1 && sync && ./build.sh "$(cat VERSION)-custom"
```
Comment (michaelshobbs) [inline Dockerfile]: does this work without the sleep?

Comment (billimek) [reply Dockerfile]: I don't think it does (consistently); [this build](https://hub.docker.com/r/billimek/logspout/builds/bhcbrrbwoeijrnbhhgznj5z/) failed but [this one](https://hub.docker.com/r/billimek/logspout/builds/bg7ymtnwbfamszz7cxsu7hp/) succeeded.

While the sleep + sync [succeeded](https://hub.docker.com/r/billimek/logspout/builds/brwm76fgojhxgbq6knyeuc4/).

I figured it is safer to do the `sleep 1` + sync than not.

Comment (michaelshobbs) [reply Dockerfile]: Any reason to not chain all of these commands with `&&`?

Comment (billimek) [reply Dockerfile]: No reason, and pushed a commit to make all commands chained with `&&`

**Comment by tobiasfielitz** [comment]
I am running these changes, works niceley.

**Pull request merged**